iis-wf/iis-wf-import/src/main/java/eu/dnetlib/iis/wf/importer/infospace/converter/DocumentToProjectConverter.java

```diff
@@ -0,0 +1,24 @@
+package eu.dnetlib.iis.wf.importer.infospace.converter;
+
+import com.google.common.base.Preconditions;
+
+import eu.dnetlib.data.proto.OafProtos.OafRel;
+import eu.dnetlib.iis.importer.schemas.DocumentToProject;
+
+/**
+ * Result-project {@link OafRel} relation to {@link DocumentToProject} converter.
+ * 
+ * @author mhorst
+ *
+ */
+public class DocumentToProjectConverter implements OafRelToAvroConverter<DocumentToProject> {
```

Comment:
Right, I see your point. Following naming conventions here (schema name + `Converter`) is definitely confusing.

I've changed it also in `ProjectToOrganizationConverter` and `ProjectToOrganizationConverterTest`.